hooks/addon

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+set -eu
+vault="secret/$GENESIS_VAULT_PREFIX"
+target=$GENESIS_ENVIRONMENT
+
+
+list() {
+  echo "The following addons are defined:"
+  echo
+  echo "  list            List out all supported addons."
+  echo "  visit-prom      Visit the Prometheus Web UI (macOS only)"
+  echo "  visit-grafana   Visit the Grafana dashboard (macOS only)"
+  echo "  visit-alert     Visit the AlertManager dashboard (macOS only)"
+  echo
```

Comment:
Did we not get an addon for generating the node_exporter runtime configs?

Reply:
Great idea. Will add.

Reply:
You can steal the one from the SHIELD kit for ... err ... inspiration.